<launch>
    <node pkg="dji_osdk_ros" type="dji_vehicle_node" name="vehicle_node" output="screen">
    <!-- node parameters -->
    <param name="acm_name" type="string" value="/dev/ttyACM0"/>
    <param name="serial_name" type="string" value="/dev/ttyTHS2"/>
    <param name="baud_rate" type="int" value="1000000"/>
    <param name="app_id" type="int" value="1076728"/>
    <param name="app_version" type="int" value="1"/>
    <param name="align_time" type="bool" value="false"/>
    <param name="enc_key" type="string" value="5ee1c5614a147f80c183fe01b17437c041af25a4de349765bd2e55139d7f50ef"/>
    <param name="use_broadcast" type="bool" value="false"/>
    </node>

    <!--
    <node pkg="dji_osdk_ros" type="record.py" name="record" output="screen">
    </node>
    <node pkg="dji_osdk_ros" type="zigbee.py" name="zigbee" output="screen">
    </node>

   

    <node pkg="mavlink_bridge" type="test.py" name="test" output="screen">
    </node>
    <node pkg="dji_osdk_ros" type="iusl_img_manage" name="iusl_img_manage" output="screen">
    </node>
    <node pkg="dji_osdk_ros" type="estimate.py" name="estimate" output="screen">
    </node>


    <node pkg="dji_osdk_ros" type="control.py" name="control" output="screen">
    </node>
    
    <node pkg="dji_osdk_ros" type="iusl_img_manage" name="iusl_img_manage" output="screen">
    </node>
    --> 









    
    
</launch>
